### DISCUSSION TITLE
Clicking on elapsed playing time in the playbar shows time remaining

### DISCUSSION THREAD

**Discussion by takaji10** [TT-ReBORN/Georgia-ReBORN#28] [Q&A] [answered]
I've seen other players use this option, and it's quite useful for some users.

Here:
![image](https://user-images.githubusercontent.com/1534470/126347764-da1ec35a-90a7-44a9-8c7c-49aefcf49c66.png)

The 3:07 would change to -0:19 or whatever amount is left on the track.

**Comment by TT-ReBORN** [comment, marked as answer]
This has been implemented, should be working now.
